R/ResetModule.R

```diff
@@ -25,14 +25,12 @@ reset_report_button_ui <- function(id, label = NULL) {
     shiny::singleton(
       shiny::tags$head(shiny::includeCSS(system.file("css/custom.css", package = "teal.reporter")))
     ),
-    shiny::actionButton(
-      ns("reset_reporter"),
-      class = "teal-reporter simple_report_button clear-report btn-warning",
-      title = "Reset",
-      `data-val` = shiny::restoreInput(id = ns("reset_reporter"), default = NULL),
-      shiny::tags$span(
-        if (!is.null(label)) label,
-        shiny::icon("xmark")
+    shinyjs::disabled(
```

Comment:
Doesn't seem to be needed as on the server side there is an observer which toggles this class. It was a problem before my changes when UI was initialized asynchronously (on the server side so that observer was trying to toggle something which didn't existed yet)

Please apply also in other button_ modules
```suggestion
```